### PR TITLE
Add other gems explanation.

### PR DESCRIPTION
**Pull request by JonRowe**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -140,6 +140,12 @@ by submitting a patch to this README.
 * [https://github.com/rspec/rspec-rails](https://github.com/rspec/rspec-rails)
 
 ## Other gems
+These gems were extracted from past versions of RSpec in order to maintain 
+compatibility for older spec suites upgrading to new versions, in particular
+`rspec-its` adds back the `it(:thing)` style of spec, `rspec-collection_matchers`,
```
Comment (pirj) [inline README.markdown]: `it(:thing)` -> `its(:thing)`

Comment (pirj) [inline README.markdown]: Extra comma after `rspec-collection_matchers`.

Comment (JonRowe) [inline README.markdown]:
```suggestion
`rspec-its` adds back the `its(:thing)` style of spec, `rspec-collection_matchers` 
```

File: README.markdown [hunk 2]
```diff
@@ -140,6 +140,12 @@ by submitting a patch to this README.
 * [https://github.com/rspec/rspec-rails](https://github.com/rspec/rspec-rails)
 
 ## Other gems
+These gems were extracted from past versions of RSpec in order to maintain 
+compatibility for older spec suites upgrading to new versions, in particular
+`rspec-its` adds back the `its(:thing)` style of spec, `rspec-collection_matchers` 
+adds back `have(n).items` style matchers, `rspec-legacy_formatters` enables old
+formatters to operate with RSpec 3+. These gems are in general not officially
+maintained by the rspec team but may receive support and maintenance as needs require.
```
Comment (benoittgt) [inline README.markdown]: It is not `RSpec team`?

Comment (JonRowe) [reply README.markdown]: Fair point i'll change it

**Pull request merged**